easyInterface/Diffraction/Interface.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, interface: Interface, app: App, calculator: Calculator, info:
         self._log.debug('Created a project dictionary')
 
     @classmethod
-    def default(cls) -> 'LoggedUndoableDict':
+    def default(cls):
```

Comment:
This type hint is less obvious at it is in principle the same as __init__, but uses an explicit return. Why was this hint removed?

Reply:
It wasn't returning a `'LoggedUndoableDict'` replaced with `'ProjectDict'`